config/jenkins-jobs_infra.ci.jenkins.io.yaml

```
@@ -147,6 +147,8 @@ jobsDefinition:
       infra-reports:
         jenkinsfilePath: Jenkinsfile_updatecli
         disableTagDiscovery: true
+      contributor-spotlight:
+        disableTagDiscovery: true
```

This setup uses `Jenkinsfile` as default. But we don't want to rebuild the whole [`Jenkinsfile`](https://github.com/jenkins-infra/contributor-spotlight/blob/main/Jenkinsfile) right?

As such, I propose you to use the same pattern as other updatecli jobs:

```suggestion
      contributor-spotlight:
        jenkinsfilePath: Jenkinsfile_updatecli
        disableTagDiscovery: true
```

THen, once merged, no build will be run for this new job until you add a `Jenkinsfile_updatecli` in a PR (like... https://github.com/jenkins-infra/contributor-spotlight/pull/146 :) )

---

Thanks!